### PR TITLE
Updated Italian translation

### PR DESCRIPTION
**Pull request by meliurwen**
### Changelog :it: 
- :heavy_check_mark: Added translation to new strings
- :wrench: Improved translation on a couple of already translated ones

### CODE REVIEW

File: po/it.po [hunk 1]
```diff
@@ -260,9 +262,8 @@ msgid "Translation updates"
 msgstr "Aggiornamenti di traduzione"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:63
-#, fuzzy
 msgid "Updated Catalán and Spanish translations thanks to Mario Rodrigo"
-msgstr "Traduzioni Catalane aggiornate grazie a Mario Rodrigo."
+msgstr "Traduzioni Catalane aggiornate grazie a Mario Rodrigo"
```
Comment (fitojb) [inline po/it.po]: The Italian doesn’t match the English here.

File: po/it.po [hunk 2]
```diff
@@ -511,10 +511,8 @@ msgid "salut! French translations thanks to Nathan (@NathanBnm)"
 msgstr "salut! Traduzioni Francesi grazie a Nathan (@NathanBnm)"
 
 #: data/com.github.cassidyjames.ephemeral.appdata.xml.in:217
-msgid ""
-"Olá! Brazilian Portuguese translations thanks to Lucas Sanchez dos Anjos"
-msgstr ""
-"Olá! Traduzioni in Portoghese Brasiliano grazie a Lucas Sanchez dos Anjos"
+msgid "Olá! Brazilian Portuguese translations thanks to Lucas Sanchez dos Anjos"
+msgstr "Olá! Traduzioni in Portoghese Brasiliano grazie a Lucas Sanchez dos Anjos"
```
Comment (fitojb) [inline po/it.po]: Language names should be written in lowercase in Italian.

File: po/it.po (hunk 1 above)
Comment (meliurwen) [reply po/it.po]: You're right! I didn't notice the difference with an another almost identical line. 😅

File: po/it.po (hunk 2 above)
Comment (meliurwen) [reply po/it.po]: True, I made I've coosed to keep the upper-case letter as a way to highlight the word in the sentence. Fixing this asap.

**Pull request merged**